Ginger/GingerCoreNET/Run/RunListenerLib/ExecutionLoggerHelper.cs

```diff
@@ -96,7 +96,8 @@ public string GetLoggerDirectory(string logsFolder)
                     if (WorkSpace.Instance != null && WorkSpace.Instance.Solution != null)
                     {
                         //If the path configured by user in the logger is not accessible, we set the logger path to default path
-                        logsFolder = WorkSpace.Instance.TestArtifactsFolder;                        
+                        //logsFolder = WorkSpace.Instance.TestArtifactsFolder;                        
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4431857)